### PR TITLE
Feat: Render SNS Neuron age

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Show the neuron's age for SNS neurons.

# Changes

* Implement the code in SnsNeuronAge that was commented.
* New sns neuron util `neuronAge`.

# Tests

* New page object SnsNeuronAge.
* New test file for SnsNeuronAge component.
* Add tests cases to SnsNeuronMetaInfoCard spec file
* Add methods to SnsNeuronMetaInfoCard page object.
* Add tests for new sns neuron util `neuronAge`.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Logic of how age bonus is calculated can be found [here](https://gitlab.com/dfinity-lab/public/ic/-/blob/07ce9cef07535bab14d88f3f4602e1717be6387a/rs/sns/governance/src/neuron.rs#L415).

Neuron is unlocked:
Screenshot 2023-06-15 at 07 47 30

Neuron is locked:
Screenshot 2023-06-15 at 07 48 04

Neuron is dissolving:
Screenshot 2023-06-15 at 07 45 30

**Comment by lmuntaner** [comment]
@mstrasinskis @peterpeterparker please review

File: frontend/src/tests/page-objects/SnsNeuronAge.page-object.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      element: this.root,
+      testId: "sns-neuron-age",
+    });
+  }
+
+  getNeuronAge(): Promise<string> {
+    return this.getKeyValuePairPo().getValueText();
+  }
+
+  hasAge(): Promise<boolean> {
```
Comment (mstrasinskis) [inline frontend/src/tests/page-objects/SnsNeuronAge.page-object.ts:23]: nitpick: `ageIsPresent`

**Review by mstrasinskis — APPROVED** [review]
Thank you, LGTM.

File: frontend/src/tests/page-objects/SnsNeuronAge.page-object.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/page-objects/SnsNeuronAge.page-object.ts]: thanks, I like the rename!

**Pull request merged**